<launch>

    <arg name="working_dir" default="$(find lfd_interface)/data" />

    <arg name="name" default="testdemo"/>

    <include file="$(find lfd_interface)/launch/lfd_storage.launch"/>
    
    <node name="lfd_trainer_node" pkg="lfd_interface" type="lfd_trainer_node" output="screen">
        <param name="demonstration_name" value="$(arg name)"/>
    </node>

</launch>
